bookkeeper-server/src/main/java/org/apache/bookkeeper/bookie/EntryLogger.java

```diff
@@ -493,6 +494,12 @@ void createNewLog() throws IOException {
         currentDir = logChannel.getLogFile().getParentFile();
     }
 
+    /**
+     * get EntryLoggerAllocator, Just for tests.
+     */
+    protected EntryLoggerAllocator getEntryLoggerAllocator() {
```

Comment:
nit: you can remove `protected` to make this method package private, because the test and class are in the same package.